src/new-components/columns/columns.test.tsx

```diff
@@ -274,19 +274,18 @@ describe('Column', () => {
     it('is set to the specified width', () => {
         const { rerender } = render(
             <Columns>
-                <Column data-testid="column" width="content" />
+                <Column data-testid="column" />
             </Columns>,
         )
         const column = screen.getByTestId('column')
 
-        // for width="content" no css class is added
-        for (const width of columnWidths) {
-            expect(column).not.toHaveClass(`columnWidth-${width.replace('/', '-')}`)
-        }
+        // // for width="content" no css class is added
+        // for (const width of columnWidths) {
+        //     expect(column).not.toHaveClass(`columnWidth-${width.replace('/', '-')}`)
+        // }
```

Comment:
💭 Is this comment meant to be kept around?

Reply:
Nope. Good catch.